staging/src/k8s.io/client-go/tools/leaderelection/leaderelection_test.go

```diff
@@ -753,6 +755,41 @@ func testReleaseOnCancellation(t *testing.T, objectType string) {
 	}
 }
 
+func TestLeaderElectionConfigValidation(t *testing.T) {
+	resourceLockConfig := rl.ResourceLockConfig{
+		Identity: "baz",
+	}
+
+	lock := &rl.LeaseLock{
+		LockConfig: resourceLockConfig,
+	}
+
+	lec := LeaderElectionConfig{
+		Lock:          lock,
+		LeaseDuration: 15 * time.Second,
```

Comment:
I'm curious if there are reasons for choosing the 3 time durations like this? Or are they just arbitrary?

Reply:
They are just arbitrary. Copy pasta from some other test. I can adjust them if you have concerns/suggestions

Reply:
No worries. I was just curious where the numbers came from. All good.